src/NorthwindWebsite/API/Controllers/CategoriesController.cs

```diff
@@ -23,4 +26,41 @@ public async Task<ActionResult<List<Category>>> GetCategories()
 
         return categories.ToList();
     }
+
+    [HttpGet("{categoryId}/image")]
```

Comment:
1. Just an advice: to avoid option to pass invalid type values you can add inline constraints, like `{categotryId:int}`
2. Honestly, `{categoryId}/image` is not really obvious, I mean, passing some kind of `categories/2/image` is not really clear if you are not familiar with app. To avoid such  misunderstandings I advice also to add some prefix that specifies searching criteria: `category/id/2/image`. Template will look like `id/{categoryId}/image`